### DISCUSSION TITLE
Pinocchio with Anaconda 32bit on Windows

### DISCUSSION THREAD

**Discussion by Richi90** [stack-of-tasks/pinocchio#1613] [Q&A] [answered]
Hello everyone, I'm trying to install the Pinocchio library  within Anaconda 32 bit on Windows 10. 
I'm having some trubles in finding the package from the channel. 
Am I missing something?
Is it possible to install Pinocchio with these requirements?
Do you have any advices?
Thank you.
Riccardo

**Comment by jcarpent** [comment, marked as answer]
You just need to do `conda install pinocchio -c conda-forge`.

**Comment by traversaro** [comment]
Sorry, I just saw this. pinocchio conda packages are provided on conda-forge, and conda-forge does not provide packages for 32-bit Windows, just 64-bit . See https://github.com/conda-forge/conda-forge.github.io/issues/946#issuecomment-569436955 for example.
